llama_stack/distribution/inspect.py

```diff
@@ -62,8 +62,8 @@ async def list_routes(self) -> ListRoutesResponse:
                     ret.extend(
                         [
                             RouteInfo(
-                                route=e.route,
-                                method=e.method,
+                                route=e.path,
+                                method=next(iter([m for m in e.methods if m != "HEAD"])),
```

Comment:
this is confusing, why is this needeed?

Reply:
because fastapi adds "HEAD" in methods if the method is GET :/